shared-helpers/src/views/CustomIconMap.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react"
+import { Application, Profile } from "./CustomIcons"
+import LockClosedIcon from "@heroicons/react/24/solid/LockClosedIcon"
+import ChevronLeftIcon from "@heroicons/react/20/solid/ChevronLeftIcon"
+import Clock from "@heroicons/react/24/outline/ClockIcon"
+import HomeModernIcon from "@heroicons/react/24/outline/HomeModernIcon"
+import EnvelopeIcon from "@heroicons/react/24/outline/EnvelopeIcon"
+
+export const CustomIconMap = {
+  application: Application,
+  profile: Profile,
+  lockClosed: <LockClosedIcon />,
```

Comment:
Curious why asome icons are mapped here and others pulled in in their respective files?

Reply:
That's a good question…I probably was thinking about one-offs vs ones that get used in a number of different places. But we might want to map all of them, and also test the theory that they could easily be swapped out for new custom icons for certain jurisdictions (in a follow up PR I imagine)